environment.yml

```diff
@@ -14,6 +14,18 @@ dependencies:
   - scipy>=1.0,<1.13
   - yamale>=4.0
   - h5py>=3.5
-  - s1reader>=0.2.0
+  - s1reader>=0.2.1
   - shapely>=1.8
   - requests>=2.0
+  - scikit-image
+# CI unit test
+  - pytest
+  - pluggy
```

Comment:
Hmm this one doesn't look like it's being used by COMPASS? I think `pluggy` is the module that `pytest` created as a general framework for making plugins

Reply:
We need those dependency to make the unit test to run in specific order. So we need to keep it as part of the dependency.